### ISSUE TITLE
For 1.4.13 Content on Hover or Focus, note at bottom of Intent section should be moved up

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#3129]
The Understanding document for [Content on Hover or Focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html) has 2 notes underneath the normative text clarifying scope. These are prominent and useful. There is another note at the bottom of the Intent section which is easier to overlook, yet is just as important for context
>This SC covers content that appears in addition to the triggering component itself. Therefore, a non-visible component, like a Skip to Main link, that is made visible on keyboard focus (with no additional content beyond the trigger becoming visible) is not covered by the SC.

There was a recent report on some failures from a tester (sorry, can't locate) where someone had wrongly failed Skip to Main links, arguing they failed 1.4.13. I was pretty sure we'd covered this section, but when I went back to quickly check, I was shocked to see it wasn't covered in the notes. This is because if looking at the SC text only, only the first two note are listed. So both of us failed to look at the bottom of the intent section.

I think this makes sense to address by making the third note be at the bottom of the SC text, not the bottom of the Intent section in the Understanding document.

**Comment by patrickhlauke** [comment]
@mbgower there's a potential wrinkle here though: the two notes are part of the normative text itself. that third note isn't, and is only present in the understanding. you can probably move if physically closer to the top in the understanding, but it would be quite odd-looking, and it may start to confuse matters further for people not grokking the difference between normative notes and non-normative notes?

**Comment by patrickhlauke** [comment]
unless you're proposing making that third note (possibly with a slight rewording) normative as well, which would be a pain to do process-wise

**Comment by mbgower** [comment]
My understanding is that none of the notes below the normative text are normative. My recollection is that we've changed that without the need for a normative change.
@alastc has put an erratum on it, which I guess will clarify this.

**Comment by patrickhlauke** [comment]
> none of the notes below the normative text are normative

oh wow, i missed that. in which case, are you suggesting adding the third note to the actual SC itself (so it also shows in the main WCAG spec, not just understanding)?

**Comment by patrickhlauke** [comment]
ah, i see https://github.com/w3c/wcag/pull/3130 does indeed move the note directly to the SC
